signer/plugin.go

```diff
@@ -86,11 +94,13 @@ func (s *pluginSigner) generateSignature(ctx context.Context, desc ocispec.Descr
 			keySpec:      ks,
 		},
 	}
-
+	logger.Debugf("generated genericSigner %+v", genericSigner)
```

Comment:
It might not be a good idea to print `genericSigner` as it has pointers.

Reply:
removed.

Reply:
This line is still there in the current status of this PR.

Reply:
Updated.